README.md

```diff
@@ -154,7 +155,7 @@ My mother. Let's play to the end together ❤️ with 🧠
 
 ## ❓ FAQ <a name="faq"></a>
 
-- **Is there any restriction to adapting the page to other porpouse??**
+- **Is there any restriction to adapting the page to other porpuse??**
 
   - No problem.
 
```

Comment:
- [ ] [OPTIONAL] Kindly check the MIT license link on line `166`. Try using the following. 😎️

```
This project is [MIT](./licence) licensed.
```